src/engine/test/scripts/schemas/generate_schemas.py

```diff
@@ -0,0 +1,174 @@
+import requests
```

Comment:
Im having doubts on leaving tis script on the repo, or at least moving it to another directory

Reply:
why?